src/app/components/LimitIE9Comment/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+/*
+ * These, when used together, stop anything within them from loading on IE9 and below.
+ */
+
+export const IE9LimitOpen = () => (
+  <div dangerouslySetInnerHTML={{ __html: '<!--[if !IE]><!-->' }} /> // eslint-disable-line react/no-danger
+);
+
+export const IE9LimitClose = () => (
+  <div dangerouslySetInnerHTML={{ __html: '<!--<![endif]-->' }} /> // eslint-disable-line react/no-danger
```

Comment:
Any reason we didn't make this generic? Could be a neat little component.

```jsx
export const HTMLComment = ({ children }) => (
  <div dangerouslySetInnerHTML={{ __html: `<!--${children}-->` }} />
);
```

Usage:

```jsx
<HTMLComment>[if !IE]><!</HTMLComment>
  <script>bla</script>
<HTMLComment><![endif]</HTMLComment>

<footer>
  <HTMLComment>Reading this? Join the BBC! http://careerssearch.bbc.co.uk/ </HTMLComment>
</footer>
```

Reply:
I think we can expand it if we decide to that, but it seems a stretch IMO and having the specific IE comment syntax in a file like `Document/component.jsx` doesn't seem great IMO

Reply:
It feels better than making a specific IE component IMO!

Reply:
Chatted IRL, I think we will leave it as it is due to desire to reduce brittleness and unique use case